subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1486,6 +1483,17 @@ void ll_rx_mem_release(void **node_rx)
 #endif /* CONFIG_BT_CTLR_SYNC_ISO */
 #endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
 
+#if defined(CONFIG_BT_CTLR_DF_SCAN_CTE_RX) || defined(CONFIG_BT_CTRL_DF_CONN_CTE_RX)
+		case NODE_RX_TYPE_SYNC_IQ_SAMPLE_REPORT:
+		case NODE_RX_TYPE_CONN_IQ_SAMPLE_REPORT:
+		{
+			ull_iq_report_link_inc_quota(1);
+			ull_df_iq_report_mem_release(rx_free);
+			ull_df_rx_iq_report_alloc(1);
+		}
```

Comment:
Either
```suggestion
			ull_iq_report_link_inc_quota(1);
			ull_df_iq_report_mem_release(rx_free);
			ull_df_rx_iq_report_alloc(1);
```
or
```suggestion
		{
			const uint8_t report_cnt = 1U;
			
			ull_iq_report_link_inc_quota(report_cnt);
			ull_df_iq_report_mem_release(rx_free);
			ull_df_rx_iq_report_alloc(report_cnt);
		}
```

Reply:
Neither of these. I leave it as is.
Braces are left to follow convention in this function.
Const variable is not necessary here. In my humble opinion `ull_iq_report_link_inc_quota` and `ull_df_rx_iq_report_alloc` with single argument being integer is enough to understand meaning of provided values.

Reply:
Ok :) My rule of thumb is that if a literal value is used more than once for the same value (e.g. number of reports), then always put in a variable. Not only does it make it easier to understand that the value `1` in both function calls is the same value, but it also makes it easier to refactor.

Given that it is a const, it will also likely be optimized by the compiler to be the same result, so I usually see such constant values as a win-win :) 

It is OK to keep as is, although I am curious to when you would use a constant instead of literals: 3 or more function calls, or never?

Reply:
Your argument about multiple use of a value with the same meaning convinced me. I've applied suggested change.